harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -268,7 +268,10 @@ def _average_training_metrics(
         self, per_batch_metrics: List[Dict[str, Any]]
     ) -> List[Dict[str, Any]]:
         """Average training metrics across GPUs"""
-        check.true(self.hvd_config.use, "Can only average training metrics in multi-GPU training.")
+        check.true(
```

Comment:
same as before on replacing `check.true` with an assertion.  If you are touching this line, please make this update as well.